utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -19,9 +20,56 @@ export function Search() {
     if (param) { setQuery(param); }
   };
 
+  const checkSkipLink = () => {
+    const link = document.getElementById('skip-link');
+    if (link) {
+      setSkipLink(link);
+    } else {
+      // Create observer to watch for new nodes
+      const obs = new MutationObserver(() => {
+        const link = document.getElementById('skip-link');
+        if (link) {
+          setSkipLink(link);
+        }
+      });
+      // @ts-ignore
+      if (!observer) setObserver(obs);
+    }
+  };
+
+  const setSkipLink = (/** @type {HTMLElement} */ link) => {
+    // Update skip link
+    link.onclick = (e) => {
+      e.preventDefault();
+      // Focus on search input
+      const content = document.getElementById('searchInput');
+      content?.focus();
+    };
+    // Disconnect observer when done
+    // @ts-ignore
+    if (observer) observer.disconnect();
+  };
+
+  useEffect(() => {
+    if (observer) {
+      // @ts-ignore
+      observer.observe(document, { subtree: true, childList: true });
+    }
+  }, [observer]);
+
   useEffect(() => {
```

Comment:
for readability, especially if the dependency array grows, the useEffect style has generally been:
```
useEffect(
  () => {
      ...
  },
  []
);
```
where the function and the dependencies are each on their own lines. do we like this one way or the other? or don't care?